web/text_layer_builder.js

```diff
@@ -53,6 +54,17 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
   }
 
   TextLayerBuilder.prototype = {
+    _afterRender: function TextLayerBuilder_afterRender() {
+      if (!this.renderingDone) {
```

Comment:
you can remove this. or make it 'finishRendering' and assign `this.renderingDone = true;` here